Makefile

```diff
@@ -1,7 +1,9 @@
 pkgs = $(shell go list ./... | grep -v /vendor/)
 files = $(shell find . -path ./vendor -prune -o -name '*.go' -print)
+ingress_binary = $(GOPATH)/bin/feed-ingress
+template = ./ingress/nginx.tmpl
 
-all : format vet lint test build
+all : format vet lint test build docker
```

Comment:
I think `docker` should be part of a release target, separate from the default (which should be for the normal dev build cycle).